src/runtime/virtcontainers/container.go

```diff
@@ -656,6 +658,8 @@ func (c *Container) createBlockDevices(ctx context.Context) error {
 						continue
 					}
 					c.mounts[i].FSGroupChangePolicy = volume.FSGroupChangePolicy(value)
+				case sensitiveMountOptions:
```

Comment:
I am surprised this is not volume.sensitiveMountOptions - as the similar options above it.

Reply:
Makes sense to move it to the right location.

Reply:
The volume const comes from direct volume package, a different go module. 
"github.com/kata-containers/kata-containers/src/runtime/pkg/direct-volume"

Kept it here only because we should be removing it later on. 
Alibaba passes the sealed secrets through different options.
https://github.com/kata-containers/kata-containers/pull/7965/files

Reply:
Understood that we will remove it later. But will suggest making the `const` part of direct volume package for now since the loop parses the `metadata` field of `mountInfo` struct which is part of the same package.